book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,146 @@
+| [<span style="color:red">zebra-script</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-script) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-script) |
+| [<span style="color:red">zebra-state</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebra-state) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebra-state) |
+| [<span style="color:red">zebrad</span>](https://github.com/ZcashFoundation/zebra/tree/main/zebrad) | [1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/tree/v1.0.0-rc.0/zebrad) |
+
+## Zcash/ZF dependencies
+
+| Name | Version | Audited | Notes
+|------| --------|-------- | -----
+| [ed25519-zebra](https://github.com/ZcashFoundation/ed25519-zebra) | [3.0.0](https://github.com/ZcashFoundation/ed25519-zebra/releases/tag/3.0.0) | |
+| [equihash](https://github.com/zcash/librustzcash) | [0.2.0](https://github.com/zcash/librustzcash/releases/tag/0.2.0) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) |
+| [halo2_proofs](https://github.com/zcash/halo2) | [0.2.0](https://github.com/zcash/halo2/tree/halo2_proofs-0.2.0) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) [mary](https://z.cash/halo2-audit/) |
+| [incrementalmerkletree](https://github.com/zcash/incrementalmerkletree) | [0.3.0](https://github.com/zcash/incrementalmerkletree/releases/tag/v0.3.0) | |  
+| [zcash_encoding](https://github.com/zcash/librustzcash) | [0.1.0](https://github.com/zcash/librustzcash/releases/tag/0.1.0) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) | 
+| [zcash_history](https://github.com/zcash/librustzcash) | 0.3.0 | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) |
+| [zcash_note_encryption](https://github.com/zcash/librustzcash) | [0.1.0](https://github.com/zcash/librustzcash/releases/tag/0.1.0) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) |
+| [zcash_primitives](https://github.com/zcash/librustzcash) | 0.7.0 | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) |
+| [redjubjub](https://github.com/ZcashFoundation/redjubjub) | [0.5.0](https://github.com/ZcashFoundation/redjubjub/releases/tag/0.5.0) | [jp](https://github.com/ZcashFoundation/redjubjub/raw/main/zcash-frost-audit-report-20210323.pdf) |
+| [orchard](https://github.com/zcash/orchard) | [0.2.0](https://github.com/zcash/orchard/releases/tag/0.2.0) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) |
+| [zcash_proofs](https://github.com/zcash/librustzcash.git?rev=ca84123038b64f0f4aa5615f7cf224fbf7ece766#ca841230) | [0.7.1](https://github.com/zcash/librustzcash.git?rev=ca84123038b64f0f4aa5615f7cf224fbf7ece766#ca841230/releases/tag/v0.7.1) | [qedit](https://hackmd.io/@qedit/zcash-nu5-audit) |
+| [zcash_script](https://github.com/ZcashFoundation/zcash_script) | [0.1.7](https://github.com/ZcashFoundation/zcash_script/releases/tag/v0.1.7) |
+
+## Cryptography dependencies
```

Comment:
@conradoplg can you check these cryptographic dependencies?

Reply:
Like I mentioned I'm not sure what criteria to use to determine if we should audit a dependency or not... Most of them are widely used, so I don't think any in particular should be audited too.

- `aes`: [audited](https://github.com/RustCrypto/block-ciphers#warnings)
- `bech32`: no audit, but seems simple enough
- `blake2b/s_simd`: no audit, but is widely used
- `bls12_381`: no audit, but seems widely used
- `bs58`: no audit, but seems simple enough
- `rand`, `rand_core`: no audits, but seems widely used
- `sha2`, `ripemd`: no audits, but seems widely used
- `secp256k1`: no audits, but seems widely used
- `subtle`: no audits, but seems widely used
- `fpe`: I think it's not being used yet
- `group`: no audits but it's just traits, seems widely used
- `x25519-dalek`: no audits, but seems widely used
- `jubjub` and `bellman`: not sure if were covered by ECC audits. Seem widely used.

Reply:
Thanks @conradoplg , i just made all the crypto dependencies out of scope in https://github.com/ZcashFoundation/zebra/pull/5395/commits/8a679e1c58e797d888e3fb1da56abfe5ad30010a based in your research. Added your comments to the list as well, we can do better but i think it is ok to just have a reason to remove them from the audit scope.